assets/js/modules/analytics/index.js

```diff
@@ -64,4 +67,45 @@ domReady( () => {
 			AREA_PAGE_DASHBOARD_ALL_TRAFFIC,
 		],
 	);
+
+	Widgets.registerWidget(
+		'analyticsGoals',
+		{
+			component: DashboardGoalsWidget,
+			width: Widgets.WIDGET_WIDTHS.QUARTER,
+			priority: 4,
+			wrapWidget: true,
+		},
+		[
+			AREA_DASHBOARD_ALL_TRAFFIC,
+		],
+	);
+
+	Widgets.registerWidget(
+		'analyticsUniqueVisitors',
+		{
+			component: DashboardUniqueVisitorsWidget,
+			width: Widgets.WIDGET_WIDTHS.QUARTER,
+			priority: 3,
+			wrapWidget: true,
+		},
+		[
+			AREA_DASHBOARD_ALL_TRAFFIC,
+			AREA_PAGE_DASHBOARD_ALL_TRAFFIC,
+		],
+	);
+
+	Widgets.registerWidget(
+		'analyticsBounceRate',
+		{
+			component: DashboardBounceRateWidget,
+			width: Widgets.WIDGET_WIDTHS.QUARTER,
+			priority: 4,
+			wrapWidget: true,
+		},
+		[
+			AREA_DASHBOARD_ALL_TRAFFIC, // this is required to have this widget appear
+			AREA_PAGE_DASHBOARD_ALL_TRAFFIC,
+		],
```

Comment:
These widget registrations are incorrect, actually already in the ACs (I've fixed that just now). They should be added to the respective `*_SEARCH_FUNNEL` widget areas instead.

Reply:
@felixarntz I've made the changes indicated however, the bounce rate only appears if I add both areas to the registration:

```js
Widgets.registerWidget(
    'analyticsBounceRate',
     {
        component: DashboardBounceRateWidget,
        width: Widgets.WIDGET_WIDTHS.QUARTER,
        priority: 4,
        wrapWidget: true,
    },
    [
        AREA_DASHBOARD_SEARCH_FUNNEL,
        AREA_PAGE_DASHBOARD_SEARCH_FUNNEL,
    ],
);
```

Reply:
@ryanwelcher I'm not sure how you mean, maybe this wasn't clear before: The Bounce Rate widget should _not_ be displayed on the dashboard, but only on the single URL details view (similar to how it is today). I just removed `AREA_DASHBOARD_SEARCH_FUNNEL` and it is working as expected for me.